third_party/move/move-compiler-v2/src/pipeline/reference_safety_processor.rs

```diff
@@ -1538,15 +1618,22 @@ impl<'env, 'state> LifetimeAnalysisStep<'env, 'state> {
             }
             // Handle case (b): check whether there is any alive mutable reference
             // which overlaps with the frozen reference.
+            let derived = self
```

Comment:
This also might be useful to create a helper function.  `self.copy_set_of_derived_refs()` or something.

Reply:
Done